<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/general/icons/Search-icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_search"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="TabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVertSys">
    <property name="title">
     <string>VertSys</string>
    </property>
    <addaction name="actionNew_Climber"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEscalador">
    <property name="title">
     <string>Escalador</string>
    </property>
    <addaction name="actionRemove_Climber"/>
    <addaction name="actionToggleActivity_Climber"/>
    <addaction name="actionPay_Climber"/>
   </widget>
   <widget class="QMenu" name="menuEmail">
    <property name="title">
     <string>Email</string>
    </property>
   </widget>
   <addaction name="menuVertSys"/>
   <addaction name="menuEscalador"/>
   <addaction name="menuEmail"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Climber"/>
   <addaction name="actionRemove_Climber"/>
   <addaction name="actionPay_Climber"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Climber">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/general/icons/New-climber.png</normaloff>:/general/icons/New-climber.png</iconset>
   </property>
   <property name="text">
    <string>Novo Escalador</string>
   </property>
  </action>
  <action name="actionRemove_Climber">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/general/icons/Rem-climber.png</normaloff>:/general/icons/Rem-climber.png</iconset>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </action>
  <action name="actionToggleActivity_Climber">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inativar</string>
   </property>
  </action>
  <action name="actionPay_Climber">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/general/icons/paying.png</normaloff>:/general/icons/paying.png</iconset>
   </property>
   <property name="text">
    <string>Pagar</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
